components/about/AdvisorList.vue

```diff
@@ -1,40 +1,42 @@
 <template>
   <div class="aboutus-table-container">
-    <h2 class="aboutus-table-title">
-      諮問一覧
-    </h2>
-    <table class="advisor-table">
-      <tbody>
-        <tr>
-          <th>諮問</th>
-          <td>田中 伸男（公共財団法人笹川平和財団 会長）</td>
-        </tr>
-        <tr>
-          <th>諮問</th>
-          <td>根元 勝則（一般社団法人日本経済団体連合会 専務理事）</td>
-        </tr>
-        <tr>
-          <th>諮問</th>
-          <td>遠藤 功（株式会社ローランド・ベルガー 日本法人会長）</td>
-        </tr>
-        <tr>
-          <th>諮問</th>
-          <td>鈴木 良伸（有限会社ステージ・フォー・ワン 代表取締役）</td>
-        </tr>
-        <tr>
-          <th>諮問</th>
-          <td>堅田 航平（五常・アンド・カンパニー株式会社 CFO）</td>
-        </tr>
-        <tr>
-          <th>諮問</th>
-          <td>渋谷 直毅（クレディ・スイス証券株式会社 インターネット・メディア部長）</td>
-        </tr>
-      </tbody>
-    </table>
+
+      <h2 class="aboutus-table-title">諮問一覧</h2>
+      <CustomTable v-bind:listdata="items" />
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6 spaces.](https://app.codacy.com/gh/AIESEC-in-Japan/aiesecjp-web/pullRequest?prid=7665075)
```suggestion
    <CustomTable v-bind:listdata="items" />
```